actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/UserBulkMigrationActor.java

```diff
@@ -318,4 +332,37 @@ private String getRootOrgId( Map<String,Object> result){
         Map<String,Object>result=((Map)((List)response.getResult().get(JsonKey.RESPONSE)).get(0));
         return result;
     }
-}
+
+    private void checkCsvHeader(List<String>csvHeaders,List<String>mandatoryHeaders,List<String>supportedHeaders){
+        checkMandatoryColumns(csvHeaders,mandatoryHeaders);
+        checkSupportedColumns(csvHeaders,supportedHeaders);
+    }
+    private void checkMandatoryColumns(List<String>csvHeaders,List<String>mandatoryHeaders){
+        ProjectLogger.log("UserBulkMigrationRequestValidator:checkMandatoryColumns:mandatory columns got "+ ArrayUtils.stringifyContents(mandatoryHeaders.toArray()),LoggerEnum.INFO.name());
```

Comment:
in logs no need to process this:ArrayUtils.stringifyContents(mandatoryHeaders.toArray()
you can directly pass list, it will print all values.